config_loader_test.go

```diff
@@ -30,13 +34,15 @@ func Test_loadFromEnvVars(t *testing.T) {
 	// these will now override the above
 	os.Setenv("ENV", "test")
 	os.Setenv("PORT", "5000")
+	os.Setenv("REDIS_REDIS_URL", "redis.bobby")
```

Comment:
You might like [`t.Setenv`](https://pkg.go.dev/testing#T.Setenv), which will clean these up when the test is over.

Reply:
making that neater